GKE: Import: Console error thrown when landing on the GKE credentials page

<!--------- For bugs and general issues --------->
**Setup**
- Rancher version: v2.11
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**
When importing a GKE cluster, a console error is thrown when landing on the GKE credentials page.
`fetch.client.js:76 Error in fetch(): TypeError: Cannot read properties of undefined (reading 'forEach')
    at Proxy.fetch (CruGKE.vue:230:20)
    at async Proxy.ca (fetch.client.js:72:3)`
<!--A clear and concise description of what the bug is.-->

**To Reproduce**
1. go to cluster management
2. click import existing
3. select Google GKE
4. notice console error
<!--Steps to reproduce the behavior-->

**Result**
When importing a GKE cluster, a console error is thrown when landing on the GKE credentials page.

**Expected Result**
Console error not thrown. 

<!--A clear and concise description of what you expected to happen.-->

**Screenshots**

<img width="1468" alt="Image" src="https://github.com/user-attachments/assets/00e8814a-e410-41a6-a795-69f385318ccc" />

<!-- If applicable, add screenshots to help explain your problem and/or errors in the browser's dev console -->

**Additional context**
found while testing https://github.com/rancher/dashboard/issues/12408
<!--Add any other context about the problem here. -->

--- a/pkg/gke/components/CruGKE.vue
+++ b/pkg/gke/components/CruGKE.vue
@@ -222,21 +222,21 @@ export default defineComponent({
       if (!this.normanCluster.gkeConfig.privateClusterConfig) {
         this.normanCluster.gkeConfig['privateClusterConfig'] = cloneDeep(defaultGkeConfig.privateClusterConfig);
       }
+      this.nodePools = this.normanCluster.gkeConfig.nodePools;
+
+      this.nodePools.forEach((pool) => {
+        pool['_id'] = randomStr();
+        pool['_isNewOrUnprovisioned'] = this.isNewOrUnprovisioned;
+        if (!pool.management) {
+          pool['management'] = {};
+        }
+        if (!pool.autoscaling) {
+          pool['autoscaling'] = {};
+        }
+      });
     }
 
     this.config = this.normanCluster.gkeConfig;
-    this.nodePools = this.normanCluster.gkeConfig.nodePools;
-
-    this.nodePools.forEach((pool) => {
-      pool['_id'] = randomStr();
-      pool['_isNewOrUnprovisioned'] = this.isNewOrUnprovisioned;
-      if (!pool.management) {
-        pool['management'] = {};
-      }
-      if (!pool.autoscaling) {
-        pool['autoscaling'] = {};
-      }
-    });
   },
 
   data() {

